src/index.js

```diff
@@ -171,24 +171,31 @@ const isHidden = function (node, displayCheck) {
 const isDisabledFromFieldset = function (node) {
   if (/^(INPUT|BUTTON|SELECT|TEXTAREA)$/.test(node.tagName)) {
     let parentNode = node.parentElement;
+    // check if `node` is contained in a disabled <fieldset>
     while (parentNode) {
+      // if it is
```

Comment:
I think this comment (and the other identical one at line 185) could be dropped. They're a bit "over the top", IMO. The others are helpful. Thank you! That will help with posterity.